tests/lib/rules/attribute-hyphenation.js

```diff
@@ -46,7 +46,7 @@ ruleTester.run('attribute-hyphenation', rule, {
     {
       filename: 'test.vue',
       code: '<template><custom data-id="foo" aria-test="bar" slot-scope="{ data }" custom-hypen="foo"><a onClick="" my-prop="prop"></a></custom></template>',
-      options: ['never', { 'ignore': ['custom-hypen'] }]
+      options: ['never', { 'ignore': ['custom-hypen', 'second-custom'] }]
```

Comment:
Can you add `second-custom` attribute in above html and also add extra failing test with same `ignore` list, but only with one attribute in HTML?

Reply:
I'm not entirely sure what you're asking for for the failing test case.  Is this what you're looking for?
```js
    {
      filename: 'test.vue',
      code: '<template><custom data-id="foo" aria-test="bar" slot-scope="{ data }" custom-hypen="foo" third-custom="bar"><a onClick="" my-prop="prop"></a></custom></template>',
      output: '<template><custom data-id="foo" aria-test="bar" slot-scope="{ data }" custom-hypen="foo" thirdCustom="bar"><a onClick="" my-prop="prop"></a></custom></template>',
      options: ['never', { 'ignore': ['custom-hypen', 'second-custom'] }],
      errors: [{
        message: "Attribute 'third-custom' cann't be hyphenated.",
        type: 'VDirectiveKey',
        line: 1
      }]
    },
```